lib/jira/commands/new.rb

```diff
@@ -2,8 +2,15 @@ module Jira
   class CLI < Thor
 
     desc "new", "Creates a new ticket in JIRA and checks out the git branch"
+    method_option :project, aliases: "-p", type: :string, default: nil, banner: "PROJECT"
```

Comment:
A user has the option for both, the long and shortened option. I did this because writing out the long option is tedious. The help menu for each command explicitly mentions this information.